lib/npg_pipeline/function/autoqc.pm

```diff
@@ -183,14 +184,17 @@ sub _create_definition_object {
     $ref->{'command_preexec'} = $self->repos_pre_exec_string();
   }
 
+  ##no critic (ControlStructures::ProhibitCascadingIfElse)
```

Comment:
Reimplement using hash at top, to avoid turning off critic

Reply:
Done